core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -448,6 +451,10 @@ class ReplicaManager(val config: KafkaConfig,
     delayedShareFetchPurgatory.tryCompleteElseWatch(delayedShareFetch, delayedShareFetchKeys)
   }
 
+  private[server] def addDelayedShareTimerRequest(timerTask: TimerTask): Unit = {
```

Comment:
The naming could be a bit tighter. There are several layers of the similar concepts, but they've all got slightly different names. `addDelayedShareFetchTimerRequest` would be more consistent here I feel.